qiime2/sdk/util.py

```diff
@@ -92,3 +92,25 @@ def parse_format(format_str):
     except KeyError:
         raise TypeError("No format: %s" % format_str)
     return format_record.format
+
+
+def has_metadata(artifact):
```

Comment:
Do we want to push this into the `Artifact` object's API? I'm fine with leaving in sdk for now and moving later on. 

pinging @jairideout @thermokarst